src/components/textarea/Textarea.tsx

```diff
@@ -1,19 +1,41 @@
-import React, { HTMLProps } from 'react';
+import React, { HTMLProps, useContext, useEffect } from 'react';
 import classNames from 'classnames';
 import { FormElementProps } from '../../util/types/FormTypes';
 import FormGroup from '../../util/FormGroup';
+import FormGroupContext from '../formgroup/FormGroupContext';
 
 type TextareaProps = HTMLProps<HTMLTextAreaElement> & FormElementProps;
 
-const Textarea: React.FC<TextareaProps> = props => (
-  <FormGroup<TextareaProps> inputType="textarea" {...props}>
-    {({ className, error, ...rest }) => (
-      <textarea
-        className={classNames('nhsuk-textarea', { 'nhsuk-textarea--error': error }, className)}
-        {...rest}
-      />
-    )}
-  </FormGroup>
-);
+const Textarea: React.FC<TextareaProps> = props => {
+  const { isInFormGroup, setInputID } = useContext(FormGroupContext);
+  useEffect(() => {
+    if (isInFormGroup && props.id) {
+      setInputID(props.id);
+      return () => {
+        setInputID(undefined);
+      };
+    }
+    return () => {};
+  }, [isInFormGroup, props.id]);
+
+  if (isInFormGroup) {
+    return <TextareaElement {...props} />;
```

Comment:
Same here, it's probably easier to just use the `<textarea>` component - it just removes a layer of abstraction and doesn't really reduce the number of lines